csatar-theme/assets/css/csatar-custom-styles.css

```diff
@@ -1845,7 +1845,7 @@ a.login:link, a.login:visited, a.login:active {
     margin-bottom: 100px;
 }
 
-[aria-labelledby="about-us-tab"] img {
+img:not(.csat-logo-img):not(.csat-data-profile-img):not(.csat-gallery-cover):not(.csat-gallery-edit-img):not(.csat-data-card img) {
```

Comment:
Ezt egy not()-al kellene megoldani, egy class-al, pl. "not-auto-height"

Reply:
Javítva